### ISSUE TITLE
Generation failure when no description defined

### ISSUE THREAD

**Issue by fpanwaskar** [gkushang/cucumber-html-reporter#7]
Error raised when generating report because feature description is not defined. 

```
lodash.templateSources[1]:54
((__t = ( feature.description.replace(/\n/g, '<br />') )) == null ? '' : __t) +
                             ^

TypeError: Cannot read property 'replace' of undefined
```

This seems to be an optional parameter (https://cucumber.io/docs/reference) and the templates need to be more defensive.

Example results: 

```
[
  {
    "elements": [
      {
        "id": "movie-list-component;vis---movie-list",
        "keyword": "Scenario",
        "line": 41,
        "name": "VIS - Movie List",
        "steps": [
          {
            "arguments": [],
            "keyword": "Before ",
            "result": {
              "status": "passed",
              "duration": 597974
            },
            "hidden": true,
            "match": {
              "location": "/features/support/Hooks.js:2"
            }
          }
        ],
        "tags": [
          {
            "name": "@aliased",
            "line": 56
          }
        ],
        "type": "scenario"
      }
    ],
    "id": "movie-list-component",
    "keyword": "Feature",
    "line": 1,
    "name": "Movie List Component",
    "tags": [],
    "uri": "/features/example.feature"
  }
]

```

**Comment by gkushang** [comment]
@fpanwaskar thanks for reporting the issue. Fixed with https://github.com/gkushang/cucumber-html-reporter/pull/8 and published as version [@0.2.2](https://www.npmjs.com/package/cucumber-html-reporter)

**Comment by gkushang** [comment]
Closing this issue since it was fixed!
